packages/web/config/generate-cosmos-kit-wallet-list.ts

```diff
@@ -79,7 +81,7 @@ async function generateCosmosKitWalletList() {
     w.logo = ""; // We'll override the logos in wallet-registry.ts
     acc[w.name] = w;
     return acc;
-  }, {} as Record<string, Wallet>);
+  }, {} as Record<string, Wallet | any>);
```

Comment:
Any reason why we are adding `any` here? I believe every registry object should match the Wallet type

Reply:
cc @isreallee82

Reply:
@JoseRFelix, it appears that the mismatched types are causing the build to fail. Although upgrading wallet-core resolves this issue, it introduces other type mismatches.